modules/ROOT/pages/concepts.adoc

```diff
@@ -1,10 +1,10 @@
-Label:: Within a Workspace, you can define a set of labels which are aliases pointing to either a branch, tag or a specific commit in your git repository. This allows you to have multiple versions of policies active at the same time. For example, you could have a `production` label that points to the trunk and a `staging` label that points to your work-in-progress branch. On every push to the git repository, Cerbos Cloud determines the set of labels affected by that push and builds bundles for each of them. Each one of your on-premise PDPs must be configured with the name of a label to watch. Cerbos Cloud takes care of distributing the correct bundles to each PDP based on the label they are configured to watch.The set of labels must be defined in a file named `.cerbos-cloud.yaml` stored in the root of the default branch you defined when setting up the workspace.
-Policy bundle:: An encrypted file containing optimised binary representations of policies corresponding to a git commit. On every commit to the policy repository, if the git reference of the commit matches a configured label, Cerbos Cloud validates the policies in the new commit, runs tests if there are any and produces a policy bundle that is then pushed to all connected PDPs that are configured to watch that label.
-Client credentials:: The PDPs establish an authenticated connection to Cerbos Cloud using a client ID and a secret. You can create a client credential using the settings section of Cerbos Cloud.
+Workspace secret:: When creating a new workspace, Cerbos Hub generates a public/private key pair and presents you with the private key. This key is not saved on Cerbos Hub servers and will be shown only once. You must copy and store it securely. All your Workspace artifacts are encrypted using public key cryptography and they can only be decrypted using the secret key. If you lose the secret key, you have to generate a new one using the Settings menu of Cerbos Hub. Generating a new key will trigger a rebuild of all your bundles. Any ADPs using the old key will stop working when the new key becomes active. You have to update the ADPs with the new key to make them work again.
+Authorization Decision Point (ADP):: The open source Cerbos server instances that you run in your own infrastructure are called ADPs. Cerbos Hub is the management control plane for ADP instances that are running inside your environment. Rather than each ADP being responsible for detecting policy changes, parsing, compiling and loading them, they get pre-compiled policy bundles pushed to them from Cerbos Hub. This model ensures that all your data remains within your network perimeter and that authorization checks happen locally with low latency while reducing the overhead of policy updates and the time it takes for the whole fleet to get in sync. A ADP must be configured with the name of a label, workspace secret and API key in order to connect to Cerbos Hub.
```

Comment:
```suggestion
Authorization Decision Point (ADP):: The open source Cerbos server instances that you run in your own infrastructure are called ADPs. Cerbos Hub is the management control plane for ADP instances that are running inside your environment. Rather than each ADP being responsible for detecting policy changes, parsing, compiling and loading them, they get pre-compiled policy bundles pushed to them from Cerbos Hub. This model ensures that all your data remains within your network perimeter and that authorization checks happen locally with low latency while reducing the overhead of policy updates and the time it takes for the whole fleet to get in sync. An ADP must be configured with the name of a label, workspace secret and API key in order to connect to Cerbos Hub.
```